tests/components/hassio/common.py

```diff
@@ -44,7 +44,7 @@ def mock_addon_store_info(
         side_effect=addon_store_info_side_effect,
     ) as addon_store_info:
         addon_store_info.return_value = {
-            "available": False,
+            "available": True,
```

Comment:
Does this impact test coverage somehow in any of the integrations?

Reply:
No, it does not. I tested we the other integration tests.

Reply:
Ok.